r2/r2/models/builder.py

```diff
@@ -173,6 +173,15 @@ def wrap_items(self, items):
             except AttributeError:
                 pass
 
+            if isinstance(item, Comment):
+                if not hasattr(w, 'subreddit'):
+                    w.subreddit = item.subreddit_slow
+                if not hasattr(w, 'link'):
+                    w.link = item.link_slow
+                if item.author_id == w.link.author_id and not w.link._deleted:
```

Comment:
I got weird results with just assuming the attributes were there; sometimes they were, sometimes they weren't; so just in case they are not, they'd be built.